rudder-agent/SOURCES/rudder.init

```diff
@@ -69,8 +74,6 @@ if [ "${3}" -eq 1 ]; then
     printf "${GREEN}OK${NORMAL}\n"
   fi
 
-else
-  echo "* ${1}: Role is not present on this machine, skipped."
```

Comment:
This is unneeded, and spams the user multiple times with the same message every time a user restarts the service:

Exemple on an agent only machine:
```
[root@centos-6-64 ~]# ./rudder.init restart
Stopping Rudder services:
 * rudder-slapd: Role is not present on this machine, skipped.
 * postgresql: Role is not present on this machine, skipped.
 * rudder-jetty: Role is not present on this machine, skipped.
 * rudder-agent...	OK

Starting Rudder services:
 * rudder-slapd: Role is not present on this machine, skipped.
 * postgresql: Role is not present on this machine, skipped.
 * rudder-jetty: Role is not present on this machine, skipped.
 * rudder-agent...	OK
```

This is better:
```
[root@centos-6-64 ~]# ./rudder.init restart
Stopping Rudder services:
 * rudder-agent...	OK

Starting Rudder services:
 * rudder-agent...	OK
```